packages/twenty-server/src/modules/connected-account/standard-objects/connected-account.object-metadata.ts

```diff
@@ -43,7 +47,7 @@ export class ConnectedAccountObjectMetadata extends BaseObjectMetadata {
     description: 'The account provider',
     icon: 'IconSettings',
   })
-  provider: string;
+  provider: ConnectedAccountProvider; // field metadata should be a SELECT
```

Comment:
+1

Reply:
but we don't sync options for standard fields for now, might be hard to make it evolve